code/modules/client/preference/preferences.dm

```diff
@@ -71,6 +71,7 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 
 	/// Volume mixer, indexed by channel as TEXT (numerical indexes will not work). Volume goes from 0 to 100.
 	var/list/volume_mixer = list(
+		"1012" = 100, // CHANNEL_GENERAL
```

Comment:
Let's keep it ordered. Can you put this one at the bottom?

Reply:
Well, order in this list defines order in the mixer's interface and putting volume setting for **all sounds** on bottom seems not a good idea

Reply:
Done!